recipes/tadrep/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.9.1" %}
+
+package:
+  name: tadrep
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/oschwengers/tadrep/archive/v{{ version }}.tar.gz
+  sha256: 'd09bcff6c295bc5532573696db4ad15f00364f8c8dc53d3907d573be56c8011b'
+
+build:
+  noarch: generic
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - tadrep = tadrep.main:main
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
+  run:
+    - python >=3.8
+    - biopython >=1.80
+    - xopen >=1.5.0
+    - pyrodigal >=2.1.0
+    - blast >=2.12.0
+    - cd-hit >=4.8.1
+    - matplotlib-base >=3.7
+    - pygenomeviz >=0.4
+
+test:
+  import:
+    - Bio.SeqIO
+    - pyrodigal
+  commands:
+    - tadrep --help
+
+about:
+  home: https://github.com/oschwengers/tadrep
+  license: GPLv3
```

Comment:
can you use here please the SPDX identifier?

Reply:
done